logger/drain/drain.go

```
@@ -0,0 +1,59 @@
+package drain
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"os"
+
+	"github.com/coreos/go-etcd/etcd"
+)
+
+func GetDrain() string {
+	host := getopt("HOST", "127.0.0.1")
+
+	etcdPort := getopt("ETCD_PORT", "4001")
+	etcdPath := getopt("ETCD_PATH", "/deis/logs")
+
+	client := etcd.NewClient([]string{"http://" + host + ":" + etcdPort})
+
+	s, err := client.Get(etcdPath+"/drain", true, false)
+	if err != nil {
+		return ""
```

np, I've rebased this.
I'm still not sure what to do about the silent error - it's probably preferable to every logged line firing a "drain not configured". I can't see a better way to pull from etcd.